julietezekwe/iReporter-API
# #169288554 Dockerize iReporter-API

#### What does this PR do?
This PR ensures the application is containerized and dockerized

#### Description of Task to be completed?
N/A

#### How should this be manually tested?
- Checkout to this branch
- Ensure `Docker Desktop` is running on your machine 
- Use `docker-compose run web rails new . --force --no-deps --database=postgresql` to build the dockerized application
- Use `docker-compose up` to boot up the dockerized application
- Finally, test all the routes in the application

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#169288554](https://www.pivotaltracker.com/story/show/169288554)
